.github/workflows/run-cypress-on-pr.yaml

```diff
@@ -0,0 +1,73 @@
+name: Cypress tests on pr
+on:
+  pull_request:
+    branches: [ master ]
+    types: [opened, synchronize, reopened]
+    paths:
+      - 'frontend/**'
+      - '!frontend/stats/**'
+      - 'backend/**'
+      - '.github/workflows/run-cypress-on-pr.yaml'
+      - 'docker-compose.yml'
+      - 'Dockerfile'
+      - 'gitea/**'
+  workflow_dispatch:
+
+jobs:
+  cypress-tests:
+    name: Build environment and run e2e test
+    timeout-minutes: 25
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+
+      - name: Generate .env file
+        run: |
+          echo CYPRESS_TEST_APP=autodeploy-v3 >> .env
+          echo DEVELOP_APP_DEVELOPMENT=0 >> .env
+          echo DEVELOP_RESOURCE_ADMIN=0 >> .env
+          echo DEVELOP_BACKEND=0 >> .env
+          echo DEVELOP_DASHBOARD=0 >> .env
+          echo DEVELOP_PREVIEW=0 >> .env
+          echo GITEA_ADMIN_PASS=g9wDIG@6gf >> .env
+          echo GITEA_ADMIN_USER=localg1iteaadmin >> .env
+          echo GITEA_CYPRESS_USER=cypress_testuser >> .env
+          echo GITEA_CYPRESS_PASS=g9wDIG@6gf >> .env
+          echo GITEA_ORG_USER=ttd >> .env
+          echo POSTGRES_PASSWORD=kyeDIG@eip >> .env
+          echo COMMIT= >> .env
+          echo IGNORE_DOCKER_DNS_LOOKUP=true >> .env
+
+      - name: Build all images
+        run: |
+           docker compose build --no-cache
+
+      - name: Install node
+        uses: actions/setup-node@v4
+        with:
+          cache: 'yarn'
+
+      - name: Run setup.js script
+        run: |
+          node ./development/setup.js
+
+      - name: Wait 30 seconds
+        run: |
+          sleep 30s
```

Comment:
Er dette for å vente på at `setup.js` blir ferdig? Er det ikke noen mulighet for å vente på en tilbakemelding fra skriptet om at det er ferdig?

Reply:
Det er bare å vente. Tror at vi kan bare slette denne step siden alt er oppe før skriptet er ferdig.

Reply:
@TomasEng I deleted the stage. Works fine without any wait